test/generators/ModelGenerators.scala

```diff
@@ -100,25 +104,35 @@ trait ModelGenerators {
       } yield ProducedDocument(documentType, reference, complement)
     }
 
+  protected val countrySpecificCodes      = Seq("DG0", "DG1")
```

Comment:
Can we use the `countrySpecificCodes` on the companion object or model as a sealed trait?